tests/atomese/interpreter/InterpreterUTest.cxxtest

```diff
@@ -107,7 +107,7 @@ public:
 		std::vector<double> exp_value = {1.0, 1.0, 0.0};
 
 		// interpret
-		atomese::Interpreter interpreter(atomese::value_key);
+		atomese::Interpreter interpreter(atomese::value_key,3);
```

Comment:
You shouldn't need to add the size of the problem for every utest, it is an optional parameter isn't it.